grails-app/controllers/au/org/emii/portal/ExternalConfigReloadController.groovy

```diff
@@ -0,0 +1,30 @@
+package au.org.emii.portal
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class ExternalConfigReloadController {
+
+    static final Logger log = LoggerFactory.getLogger(this)
+
+    def grailsApplication
+
+    def index() {
+        def config = grailsApplication.config
+        def locations = config.grails.config.locations
+
+        locations.each { location ->
+            String configFileName = location.split("file:")[1]
+            log.info "Reloading config from file '${configFileName}'..."
+            config.merge(new ConfigSlurper().parse(new File(configFileName).text))
```

Comment:
I was just about to update the PR for this exact reason, since I just opened it in IntelliJ (which seems to inject that location by itself).